src/flags/FlagInteractor.ts

```diff
@@ -31,6 +34,12 @@ export async function flagRating(params: {
                 ratingId: params.ratingId,
                 flag: params.flag,
             });
+            const rating = await getRating(params.ratingId);
+            const learningObject = await getLearningObject({
+                learningObjectId: rating.source,
+            });
+            const flagNotifier: FlagNotifier = new SlackGateway();
+            await flagNotifier.sendFlagNotification(params.user.username, rating.comment, learningObject.name, learningObject.author.username);
```

Comment:
We probably shouldn't wait for this request to be sent in order to respond to the client as the request to flag shouldn't be dependent on the successful completion of this event.
It may be a good idea however, to catch and report any errors from this request.
```suggestion
             flagNotifier.sendFlagNotification(params.user.username, rating.comment, learningObject.name, learningObject.author.username).catch((e)=>{
             // report error
             });
```